staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/helpers.go

```diff
@@ -35,7 +35,7 @@ func SetCRDCondition(crd *CustomResourceDefinition, newCondition CustomResourceD
 
 	if existingCondition.Status != newCondition.Status {
 		existingCondition.Status = newCondition.Status
-		existingCondition.LastTransitionTime = newCondition.LastTransitionTime
+		existingCondition.LastTransitionTime = metav1.NewTime(time.Now())
```

Comment:
Wow, thanks for catching this, @CaoShuFeng!

I think we can make it cleaner by writing `newCondition.LastTransitionTime = metav1.NewTime(time.Now())` on the top of the func instead. Wdyt?

```diff
--- a/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/helpers.go
+++ b/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/helpers.go
@@ -26,9 +26,9 @@ import (
 // SetCRDCondition sets the status condition.  It either overwrites the existing one or
 // creates a new one
 func SetCRDCondition(crd *CustomResourceDefinition, newCondition CustomResourceDefinitionCondition) {
+       newCondition.LastTransitionTime = metav1.NewTime(time.Now())
        existingCondition := FindCRDCondition(crd, newCondition.Type)
        if existingCondition == nil {
-               newCondition.LastTransitionTime = metav1.NewTime(time.Now())
                crd.Status.Conditions = append(crd.Status.Conditions, newCondition)
                return
        }
```

Reply:
Done